quickstart build fails

In release/version 2.10, when running env.sh:
[ERROR] Failed to execute goal on project datawave-ws-security: Could not resolve dependencies for project gov.nsa.datawave.webservices:datawave-ws-security:ejb:2.10.0-SNAPSHOT: Could not transfer artifact javax.transaction:jta:jar:1.0.1B from/to boundless (https://repo.boundlessgeo.com/main/): repo.boundlessgeo.com: Unknown host repo.boundlessgeo.com -> [Help 1]
Appears repo at https://repo.boundlessgeo.com/main/ no longer exists.
Replacing repo with https://repo.osgeo.org/repository/release/ in main pom.xml fixed the issue.

--- a/pom.xml
+++ b/pom.xml
@@ -883,6 +883,11 @@
                 <version>${version.geotools}</version>
             </dependency>
             <dependency>
+                <groupId>org.hibernate</groupId>
+                <artifactId>hibernate-annotations</artifactId>
+                <version>3.4.0.GA</version>
+            </dependency>
+            <dependency>
                 <groupId>org.infinispan</groupId>
                 <artifactId>infinispan-commons</artifactId>
                 <version>${version.infinispan}</version>
@@ -1123,9 +1128,10 @@
             <snapshots>
                 <enabled>false</enabled>
             </snapshots>
-            <id>boundless</id>
-            <url>https://repo.boundlessgeo.com/main/</url>
+            <id>osgeo</id>
+            <url>https://repo.osgeo.org/repository/release/</url>
         </repository>
+        <!--
         <repository>
             <snapshots>
                 <enabled>false</enabled>
@@ -1134,6 +1140,7 @@
             <name>Open Source Geospatial Foundation Repository</name>
             <url>https://download.osgeo.org/webdav/geotools/</url>
         </repository>
+        -->
     </repositories>
     <pluginRepositories>
         <!--

